prdoc/schema_user.json

```diff
@@ -0,0 +1,228 @@
+
+          {"const": "Runtime User", "title": "Someone involved or interested in the governance process or noticeable changes to the runtime and/or its dependencies"}
+
+        ]
+      },
+      "crate": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "semver": {
+            "$ref": "#/$defs/semver"
+          },
+          "note": {
+            "type": "string"
+          }
+        }
+      },
+      "semver": {
+        "type": "string",
+        "enum": [
+          "major",
+          "minor",
+          "patch",
+          "no change"
+        ]
+      },
+      "migration_db": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "description": {
+            "type": "string"
+          }
+        },
+        "additionalProperties": false,
+        "required": [
+          "name",
+          "description"
+        ]
+      },
+      "migration_runtime": {
+        "type": "object",
+        "properties": {
+          "pallet": {
+            "type": "string"
+          },
+          "description": {
+            "type": "string"
+          }
+        },
+        "additionalProperties": false,
+        "required": [
+          "pallet",
+          "description"
+        ]
+      },
+      "doc": {
+        "type": "object",
+        "description": "You have the the option to provide different description of your PR for different audiences.",
+        "additionalProperties": false,
+        "properties": {
+          "audience": {
+            "description": "The selected audience",
+            "$ref": "#/$defs/audience"
+          },
+          "title": {
+            "type": "string",
+            "description": "Optional title override for the PR and for the current audience"
+          },
+          "description": {
+            "description": "Description of the change",
+            "type": "string"
+          },
+          "notes": {
+            "description": "Optional relevant note",
+            "$ref": "#/$defs/array_of_strings"
+          }
+        }
+      },
+      "array_of_strings": {
+        "description": "An array of strings that can be empty",
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+      },
+      "host_function": {
+        "type": "object",
+        "additionalProperties": false,
+        "description": "List of host functions and their properties",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "enabled": {
+            "type": "boolean"
+          },
+          "description": {
+            "type": "string"
+          },
+          "notes": {
+            "type": "string"
+          }
+        },
+        "required": [
+          "name",
+          "enabled",
+          "description"
+        ]
+      }
```

Comment:
Same.

Reply:
Same comment than for the `migration_db `.

What could be done however if that makes it easier is to not have `db` and `runtime` embeded under a `migrations` section but leave it flat. It may be even easier this way.

So instead of:
```
migrations:
   db: []
   runtime: []
```

we could have:
```
db_migration: []
runtime_migration: []
```